doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -282,6 +270,110 @@ Error list (HTTP response code in paranthesis):
 * UnsupportedMediaType (415). Request content type is not supported.
 * InvalidModification (422). Some piece of information is missing in payload.
 
+## Geospatial properties of entities
+
+The geospatial properties of a context entity can be represented by means of regular context attributes.
+The provision of geospatial properties enables the resolution of geographical queries.
+
+Two different syntaxes must be supported by compliant implementations: 
+
+* *Simple Location Format*. It is meant as a very lightweight format that developers and users can quickly
+and easily add to their existing entities with little effort.
+
+* *GeoJSON*.  [GeoJSON](https://tools.ietf.org/html/draft-butler-geojson-06)
+is a geospatial data interchange format based on the JavaScript Object Notation (JSON). GeoJSON provides greater flexibility
+allowing the representation of complex geospatial features.
+
+Additionally, geospatial properties might be declared by means of [civic addresses](https://schema.org/PostalAddress),
+but in that particular case context producers will have to implement their own geocoding procedures,
+so that geographical queries can be resolved.
+
+Implementations are responsible for determining which entity attributes convey geospatial properties (by inspecting
```

Comment:
I have clarified it a bit at https://github.com/jmcanterafonseca/fiware-orion/commit/e396fd3c6f10ad85e68dcdc0481dfbfd3624c397